k8s-bench/main.go

```diff
@@ -30,14 +30,30 @@ import (
 )
 
 type Task struct {
-	Goal       string `json:"goal"`
 	Setup      string `json:"setup,omitempty"`
 	Verifier   string `json:"verifier,omitempty"`
 	Cleanup    string `json:"cleanup,omitempty"`
 	Difficulty string `json:"difficulty"`
 	Disabled   bool   `json:"disabled,omitempty"`
 
 	Expect []Expectation `json:"expect,omitempty"`
+
+	Script []ScriptStep `json:"script,omitempty"`
+
+	// Isolation can be set to automatically create an isolated cluster
+	// TODO: support namespaces also
+	Isolation IsolationMode `json:"isolation,omitempty"`
```

Comment:
I think we are mixing two things here: task definition with the environment against which tasks are run.